rfcs/0002-use-dynamic-workspace-indicator-on-gnome.md

```diff
@@ -0,0 +1,47 @@
+# RFC: Use dynamic workspace indicator on GNOME
+
+* **RFC Number:** `0002`
+* **Author(s):** [Elkhan Mammadli](elkhan@almalinux.org)
+* **Status:** Draft
+* **Created:** [2025-04-02 14:00 UTC]
+* **Updated:** [2025-04-02 14:00 UTC]
+
+## Abstract
+
+Do not replace the dynamic workspace indicator with AlmaLinux OS logo.
+
+## Motivation
+
+* **Problem Statement:** [GNOME 45](https://release.gnome.org/45) introduced a [dynamic workspace indicator](https://release.gnome.org/45/activities-loop.webm) that replaced the "Activities" button. This indicator shows the number of workspaces and enables switching between them via mouse scrolling. Although the GNOME version on AlmaLinux OS 10 is 47, the dynamic workspace indicator is visually replaced by the AlmaLinux OS logo. Which means switching between the workspaces still possible via mouse scrolling on the AlmaLinux logo.
+* **Goals:** For a complete experience, the dynamic workspace indicator should be consistent with GNOME's implementation.
+
+## Detailed Design
+
+* **Proposal Details:** Remove the branding patch from the [gnome-shell](https://git.almalinux.org/rpms/gnome-shell) package.
+* **Implementation Plan:**
+    - Remove `0001-panel-Use-branding-in-activities-button.patch` from the spec file of `gnome-shell` package.
+    - Automate the removal of the patch each time package sources are synchronized.
+* **Compatibility:** Since AlmaLinux OS 10 not released yet. This change only affects the AlmaLinux OS Kitten 10.
```

Comment:
I think the right thing to do here would be to fix the patch so that the branding is injected to the _left_ of the indicator rather than outright replacing it. It should be fairly straightforward to adjust the patch to do that.

Reply:
Could we possibly not do that? I think the sans-logo variant is aesthetically superior and some users might be confused by what appears to be 2 buttons.

Reply:
This would also keep the behavior consistent with upstream GNOME.

Reply:
I worry about this breaking GNOME extensions. Previously the RHEL patch broke Dash to Panel for me and I had to submit an issue for that. https://github.com/home-sweet-gnome/dash-to-panel/issues/2289. Having both logos could cause other side effects on extension support.

Reply:
Personally, I will not consider dropping the logo as an acceptable option at all.

Reply:
Is this something that we could configure with dconf?

Reply:
Not easily. It would require reworking this patch and introducing patches to have a setting be available.